app/src/main/res/values/strings.xml

```diff
@@ -20,7 +20,7 @@
 
     <string name="itapirema">preencher seu nome completo aqui</string>
 
-    <string name="john">preencher seu nome completo aqui</string>
+    <string name="john">John Kennedy F </string>
```

Comment:
Ah sim blz. Vou corrigir aqui!

Em 28 de ago de 2016 9:44 AM, "Gabriel Bernardo Pereira" <
notifications@github.com> escreveu:

> In app/src/main/res/values/strings.xml
> <https://github.com/gabeira/Plantare/pull/3#discussion_r76532181>:
>
> > @@ -20,7 +20,7 @@
> >
> >      <string name="itapirema">preencher seu nome completo aqui</string>
> >
> > -    <string name="john">preencher seu nome completo aqui</string>
> > +    <string name="john">John Kennedy F </string>
>
> @John-Ke <https://github.com/John-Ke> voce usou o branch do @firemanxbr
> <https://github.com/firemanxbr> , o ideal é criar o seu branch e o seu
> Pull Request.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gabeira/Plantare/pull/3/files/e5b484c4f35aa8fa51d4544de74d703eaa3d0dd0..1a0b88575375b65ce11e98bef6eed546513ecefa#r76532181>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMe8sd2G4wdaeE-hDtqnhmuVebZWjVVEks5qkZDOgaJpZM4Ju7Jc>
> .
>